src/main/java/com/transferwise/openbanking/client/api/event/EventClient.java

```diff
@@ -0,0 +1,56 @@
+package com.transferwise.openbanking.client.api.event;
+
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscription1;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscriptionsResponse1;
+import com.transferwise.openbanking.client.configuration.AspspDetails;
+import com.transferwise.openbanking.client.configuration.SoftwareStatementDetails;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscriptionResponse1;
+
+/**
+ *  Interface specifying operations supported for event API open banking in 3.1.10
+ */
+public interface EventClient {
+
+    /**
+     * Create an Event Subscription :
+     * Asking a ASPSP to notify callback whenever there are updates on the given event.
+     *
+     * @param eventSubscriptionRequest  request for event contains {callback URL, event type, version}
+     * @param aspspDetails -> detail of ASPSP which has the event to subscribe.
+     * @param softwareStatementDetails ->  The details of the software statement that the ASPSP registration uses
+     * @return OBEventSubscriptionResponse1 -> Event {subscription id, callback URL, version, event types}
+     */
```

Comment:
Please follow java documentation guidelines like ` @param nameOfParam<tab>short description what it is ` - no `{`, `}` or `->`. You can check how it is written in other clients

Reply:
Fixed in [c5c5c21](https://github.com/transferwise/openbanking-client/pull/170/commits/c5c5c21782dd2bc3e4d8cd3ffeacbf56dfa329cc).